CONTRIBUTING.md

```diff
@@ -3,14 +3,14 @@
 To develop on this project, please fork the repo and clone into your `$GOPATH`.
 
 Dependencies are **not** checked in so please download those separately.
-Download the dependencies using [`dep`](https://github.com/golang/dep).
+Download the dependencies using `go mod download`.
```

Comment:
I think it would be better to use `go mod vendor` here as this downloads deps to a vendor folder in the working directory, whereas `go mod download` installs deps to `$GOPTAH/pkg/...`

@JoelSpeed @steakunderscore what do you think?

Reply:
I think `go mod vendor` is the recommended approach if you're using tooling that may not be 100% compatible with modules, I guess if the linter works though it should be fine with either

I'm happy to have either